backend/internal/handlers/classrooms/classrooms.go

```diff
@@ -13,27 +19,101 @@ func (s *ClassroomService) getUserClassrooms() fiber.Handler {
 // Returns the details of a classroom.
 func (s *ClassroomService) getClassroom() fiber.Handler {
 	return func(c *fiber.Ctx) error {
-		// Implement logic here
-		return c.SendStatus(fiber.StatusNotImplemented)
+       	classroomID, err := strconv.ParseInt(c.Params("classroom_id"), 10, 64)
+        
+        classroomData, err := s.store.GetClassroomByID(c.Context(), classroomID)
+        if err != nil {
+            return err
+        }
+
+        return c.Status(http.StatusOK).JSON(classroomData)
 	}
 }
 
 // Creates a new classroom.
 func (s *ClassroomService) createClassroom() fiber.Handler {
 	return func(c *fiber.Ctx) error {
-		// Implement logic here
-		return c.SendStatus(fiber.StatusNotImplemented)
-	}
+        var classroomData models.Classroom
+        err := c.BodyParser(&classroomData)
+        if err != nil {
+            return err
+        }
+
+        
+        createdClassroom, err := s.store.CreateClassroom(c.Context(), classroomData)
+        if err != nil {
+            return err
+        }
+
+        return c.Status(http.StatusOK).JSON(fiber.Map{
+            "message": "created classroom",
+            "classroom data" : createdClassroom,
```

Comment:
I think convention is to avoid spaces in json keys like this. I would either just call it "classroom" or "classroom_data"